src/main/resources/db/changelog/changes/create-consortium-table.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <changeSet id="MODCON-18@@create-consortium-table" author="singhadesh">
+
+    <createTable tableName="consortium">
+      <column name="id" type="uuid">
+        <constraints primaryKey="true" primaryKeyName="pk_consortium_id"/>
+      </column>
+      <column name="name" type="text">
+        <constraints unique="true" nullable="false"/>
+      </column>
+    </createTable>
+
+    <createIndex indexName="id_idx" tableName="consortium">
```

Comment:
We don't need to create index for primary key, for name also, for now it will be only 1 record in this table with name MOBIUS